### ISSUE TITLE
Can GNURoot Install the rootfs on a sdcard 

### ISSUE THREAD

**Issue by yoke88** [corbinlc/GNURootDebian#118]
because the internal storage of my android phone was too small ,i want to install the rootfs on external SDcard , i have seen there was a checkbox (install on sdcard) ,but it  gray out. so i checked the source code of gnuroot.debian ,but i have not seen usefull code to install rootfs to SDcard .

when using  external SD-card ,the SD-card default  mount with noexec and nodev param . I have noticed that Linux live CD using a unionfs ,which mount a image file and overlay the rootfs which is readonly , the image file can live on a vfat partion ,so can we do something on the rootfs and let gnuroot to support external SD-card easily

**Comment by AdamMatthewLTM** [comment]
Hi all,

I am currently assisting the developers in sorting relevant issues for GNURoot Debian on GitHub. If the issue discussed is still relevant, could you please reply to this thread and state the issue needs to be resolved and I will attempt to sort it for you ASAP. If the issue is no longer relevant, please reply to this thread and state it has been resolved, and I will suggest this thread be closed.

Furthermore, if a particular issue is considered to be high priority, please state that you are requesting it to be considered as high priority and I will mark it as such.

Thank you all so much for your time and understanding,
-Adam

**Comment by ExtremeNerd** [comment]
Confused if a workaround can be built using usermode virtualization & using virtual machine images... Maybe QEMU usermode emulation for host architecture or something similar?

**Comment by abubelinha** [comment]
@AdamMatthewLTM Yes, I think this issue is still relevant. It is the only way to install the rootfs in small phones. Thanks a lot
